api/server/cluster.go

```diff
@@ -648,6 +650,23 @@ func (c *clusterApi) eraseAlert(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode("Successfully erased Alert")
 }
 
+//TODO(ram-infrac): add swagger yaml here
+//This function does nothing right now just dump recieved api params and
+//return success when called
+func (c *clusterApi) setClusterSecretKey(w http.ResponseWriter, r *http.Request) {
+	var dcReq api.ClusterSecretKeyRequest
+	method := "setClustersecretKey"
+	if err := json.NewDecoder(r.Body).Decode(&dcReq); err != nil {
+		c.sendError(c.name, method, w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	//Logged recived paramss for now
+	fmt.Println("clusterkey", dcReq.Clustersecretkey, "override", dcReq.Override)
+
+	w.Write([]byte(secretKeyOkMsg + "\n"))
```

Comment:
Seems like these lines are just for debugging, right?

Reply:
yes Luis those are for debugging purpose, I carved out code for /setclustersecretkey endpoint and put this PR request to clarify approach.